pkg/ibmvpc/parser.go

```diff
@@ -744,6 +730,7 @@ func getTgwObjects(c *datamodel.ResourcesContainerModel,
 				},
 				vpcs:            []*VPC{vpc},
 				availableRoutes: map[string][]*ipblocks.IPBlock{},
+				region:          &Region{name: region},
```

Comment:
should use `regionToStruct` here ?
also, use `*tgw.Location` (var `region` is not always assigned)